Services/Worker/OJS.Services.Worker.Business/ExecutionContext/Implementations/ExecutionContextBuilderService.cs

```diff
@@ -43,15 +43,15 @@ public OjsSubmission<TInput> BuildOjsSubmission<TInput>(SubmissionServiceModel s
     {
         var submission = submissionServiceModel.Map<OjsSubmission<TInput>>();
 
+        submission.StartedExecutionOn = submissionServiceModel.StartedExecutionOn;
```

Comment:
Moving this does absolutely nothing to the end result. This is just copying the value from one model, to another.